packages/cxl-lumo-styles/src/icons.js

```diff
@@ -53,6 +53,9 @@ $documentContainer.innerHTML = `
           <path d="M5.614.416a4.538 4.538 0 011.824.086A4.259 4.259 0 005.614.416zM2.322 12.449l.002.009c-.747-1.19-.94-.39-.97-.224-.587-1.581-.96-2.743-.96-2.743a10.93 10.93 0 01-.15-.61L.22 8.77c-.034-.168-.063-.33-.09-.484v-.003a16.671 16.671 0 01-.077-.583C-.04 6.725.04 5.89.23 5.176c.011-.039.02-.08.034-.117.01-.034.02-.07.033-.102.335-1.039.72-1.273.72-1.273 1-.892.963.434.963.434-1.05 2.304.227 7.84.338 8.312.003.006.003.013.005.019zM3.472 1.432A4.452 4.452 0 015.24.492a4.102 4.102 0 00-1.768.94z"/>
           <path d="M1.355 12.234l-.006.029A17.108 17.108 0 01.243 8.88c.043.2.094.403.15.61 0 0 .374 1.16.962 2.743zM.054 7.7C-.064 6.576.05 5.967.05 5.967c.055-.3.116-.563.18-.79-.19.714-.269 1.549-.175 2.525z"/>
         </g>
+        <g id="new" width="20" height="20" viewBox="0 0 20 20">
```

Comment:
I think this branch needs rebase on top of `feat/dashboard`. the other branch already have "new" icon https://github.com/conversionxl/aybolit/blob/e4a46aa4647d7882365e5a79aaf7b08ce7481d93/packages/cxl-lumo-styles/src/icons.js#L56-L58